### ISSUE TITLE
Is it possible to train the ExpansionNet_v2 model in other languages (Korean)?

### ISSUE THREAD

**Issue by ckdghks** [jchenghu/ExpansionNet_v2#18]
Hello,
I am currently using your ExpansionNet_v2 model to perform image captioning in Korean. After conducting training on my own, the results, such as CIDER or BLEU scores, only reach around 0.7, indicating that the training is not progressing as expected.
Is it possible to train the ExpansionNet_v2 model on datasets in other languages, such as Korean?
 
Thank you.
![image](https://github.com/user-attachments/assets/d998cb98-09b2-4984-aa8d-7015d5a24f73)

**Comment by jchenghu** [comment]
Hi,

Yes, the training should work with any language, including Korean. 

I assume that CIDEr-D: 76.6 and BLEU 70.4 refer to the first epoch of the partial cross-entropy training (step 2 in the readme). If that is the case, then it might be the expected result, since the English version achieves around ~92 CIDEr and ~70 BLEU in the first epoch. It is normal to have a low score at the beginning, due to the large number of warm-up steps.

Differences might arise due to a different language because of how evaluation metrics (such as CIDEr) are designed, especially when comparing quite different languages such as Korean and English.

**Comment by ckdghks** [comment]
I am currently conducting research on the Image Captioning task for the first time, and I feel that I might not fully understand language evaluation. The image above shows the completion of the 6th training step. I interpreted the value of 0.7666 as it is, but now I understand that it should be interpreted as approximately 76.66. I get it now.

So, if I aim to achieve a CIDEr score of around 1.20 by adding more datasets or improving the model, does that mean the training is going well? I would appreciate your advice on this!

**Comment by jchenghu** [comment]
> I am currently conducting research on the Image Captioning task for the first time, and I feel that I might not fully understand language evaluation. The image above shows the completion of the 6th training step. I interpreted the value of 0.7666 as it is, but now I understand that it should be interpreted as approximately 76.66. I get it now.

We've decided to be consistent with the numerical form of the [original](https://github.com/tylin/coco-caption) evaluation implementation, but it is understandably confusing.

> So, if I aim to achieve a CIDEr score of around 1.20 by adding more datasets or improving the model, does that mean the training is going well? I would appreciate your advice on this!

Yes! I believe in the English language case, the model should achieve about > 120 (or 1.20) CIDEr already when training on the COCO dataset, so yeah, if you reach that score in the first training phase, also in the Korean language, the training is going well :-)
